blog/docs/steering/2021-end-user-election.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2021 Steering Committee End User Seat Election
+date: 2021-12-17
+---
+
+Hello, Knative community,
+
+We want to kick off the Knative Steering Committee (SC) End User election, and we are ready to accept nominations from End Users!
+
+This is the first time an End User is elected to become a member of the Knative Steering Committee (SC)
```

Comment:
```suggestion
This will be the first time an End User is elected to become a member of the Knative Steering Committee (SC).
```
Since the election hasn't happened yet.